### ISSUE TITLE
Invisibility

### ISSUE THREAD

**Issue by FelipeRenault** [ZerdBG3/RAW#143]
Currently, an invisible character rolls a saving throw against an enemy that has See Invisibility to remain invisible. I will absolutely remove the saving throw, since it's ridiculous for a character with See Invisibility (or any form of see invisible sight) to not be able to find an invisible character
However, it is not possible to have an invisible character for just a portion of the enemies, so I believe a good solution is to give the revealed character an intermediary status that grants the character advantage on attacks and disadvantage on being attacked against characters that lack the see invisibility, until they break their invisibility, as they would otherwise

**Comment by drkekyll** [comment]
I mean... it's actually not completely ridiculous given that you'd still need to actually look at the character to see them and they could be hiding. Were you going to account for that or does See Invisibility make you glow through solid objects now?

**Comment by FelipeRenault** [comment]
Invisibility is not hide. Even though you're invisible, by RAW, everyone knows exactly where you are because you're making noise. That is, unless you use your action to hide. Attackers still have disadvantage to attack an invisible character and they can't use spells or abilities that specify "a target you see". Invisibility provides the character the heavily obscured condition, which is the pre-requisite to attempt to hide in the first place. So all it does is allow characters to attempt to roll for hide right in the open.
However, whenever an enemy has the ability to see invisible, the rules goes back to the default hiding rules. They can see the invisible character, so they can no longer hide from the character that see invisible if they have Line Of Sight. Of course, if they took their time to hide behind cover (and roll for stealth), they'll still not be detected until they lose cover

**Comment by drkekyll** [comment]
Right. The point I was making was that you can still hide while invisible, so you shouldn't necessarily be revealed automatically because then other characters will still be able to target a character they wouldn't otherwise be able to. I didn't mean to imply there was no way to address this only that it seemed that was Larian's reasoning for not making revelation automatic.

**Comment by drkekyll** [comment]
I've been thinking about it, and unless SE can dynamically clone statuses the best way might be to make a `_DETECTED` version of the invisible statuses and apply them on remove of the normal status by detect. To ensure the correct invisibility gets used.
